requirements/README.md

```diff
@@ -16,8 +18,27 @@ pip-compile requirements/requirements_ansible.in > requirements/requirements_ans
 
 ## Known Issues
 
-* Remove the `-e` from packages of the form `-e git+https://github.com...` in the generated `.txt`. Failure to do so will result in a "bad" RPM and DEB due to the `pip install` laying down a symbolic link with an absolute path from the virtualenv to the git repository that will differ from when the RPM and DEB are build to when the RPM and DEB are installed on a machine. By removing the `-e` the symbolic egg link will not be created and all is well.
-
 * As of `pip-tools` `1.8.1` `pip-compile` does not resolve packages specified using a git url. Thus, dependencies for things like `dm.xmlsec.binding` do not get resolved and output to `requirements.txt`. This means that:
   * can't use `pip install --no-deps` because other deps WILL be sucked in
   * all dependencies are NOT captured in our `.txt` files. This means you can't rely on the `.txt` when gathering licenses.
+
+* Both pip and setuptools need to be added back into `requirements_ansible.txt`
+
+* Python3 exceptions need to be re-added back to `requirements_ansible.txt`
+
+* The pip-compile tool is known to error due to a pycurl dependency from ovirt-engine-sdk-python. Until this is resolved, you may need to manually remove the ovirt dependency before running the tool and add it back in once finished.
+
+
+## Licenses and Source Files
+
+If any library has a change to its license with the upgrde, then the license for that library
+inside of `docs/licenses` needs to be updated.
+
+For certain libraries, a tarball of the library is kept along with the license.
```

Comment:
For libraries that have source distribution requirements (LGPL as an example), a tarball...